packages/documentation/stories/components/alert/alert.stories.tsx

```diff
@@ -0,0 +1,238 @@
+import React from 'react';
+import parse from 'html-react-parser';
+import { useArgs } from '@storybook/client-api';
+import docsPage from './alert.docs.mdx';
+
+import '@swisspost/design-system-styles/basics.scss';
+import '@swisspost/design-system-styles/components/alert.scss';
+import '@swisspost/design-system-styles/components/icons.scss';
+import './alert.stories.scss';
+
+export default {
+  title: 'Components/Alert',
+  parameters: {
+    docs: {
+      page: docsPage
+    },
+    controls: {
+      exclude: [
+        'Icon',
+        'Icon Color',
+        'No Icon',
+        'Dismissible',
+        'Fixed'
+      ]
+    }
+  },
+  args: {
+    title: 'Titulum',
+    content: '<p>Contentus momentus vero siteos et accusam iretea et justo.</p>',
+    variant: 'alert-primary',
+    icon: '',
+    iconVariant: '',
+    noIcon: false,
+    dismissible: false,
+    dismiss: false,
+    fixed: false,
+    show: false
+  },
+  argTypes: {
+    title: {
+      name: 'Title',
+      control: { type: 'text' },
+      table: { category: 'Content' }
+    },
+    content: {
+      name: 'Content',
+      control: { type: 'text' },
+      table: { category: 'Content' }
+    },
+    variant: {
+      name: 'Variant',
+      control: {
+        type: 'select',
+        labels: {
+          'alert-primary': 'Primary',
+          'alert-notification': 'Notification',
+          'alert-success': 'Success',
+          'alert-danger': 'Danger',
+          'alert-error': 'Error',
+          'alert-warning': 'Warning',
+          'alert-info': 'Info'
+        }
+      },
+      options: [
+        'alert-primary',
+        'alert-notification',
+        'alert-success',
+        'alert-danger',
+        'alert-error',
+        'alert-warning',
+        'alert-info'
```

Comment:
Bootstrap provides a `secondary` alert class. At the moment, there is one type of alert we do not cover here with a light grey background (https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components?node-id=2412%3A68325&t=3Z7AnhTO6FV4qHgT-4). We could either use the secondary variant to represent the light grey alert or create another variant while defining another default for secondary alerts. This can be handled by a separate ticket.

Reply:
1. If there are new styles, we should first update the alert styles (which is not part of this task).
2. We do not have a style for alert-secondary. And we should not start mixing bootstrap and the Design-System together. Not everything that is possible in bootstrap is possible in the Design System and vice versa.

Reply:
Created a new Ticket #827